linode/manager
# fix: Dark Mode style regressions

## Description 📝

- I caused some minor dark mode regressions in https://github.com/linode/manager/pull/11081
- This fixes the regressions by merging the theme options **before** calling `createTheme `

## Related Docs 📖 
See https://v5.mui.com/material-ui/customization/theming/#createtheme-options-args-theme

I think the key point is

> Only the first argument (options) is processed by the createTheme function.

Calling `createTheme` before merging the raw options has implications that I didn't foresee. Calling `createTheme` with the theme options merged first fixes the issues we are seeing

## Screenshots of the Issues 📷

<img src="https://github.com/user-attachments/assets/2b2f896e-7a1a-4e7b-8e99-f00e80aeef53"  height="300px" />

![Screenshot 2024-10-17 at 2 04 37 PM](https://github.com/user-attachments/assets/1b8a4f5f-2d8d-467d-a68c-13072aafed77)
![Screenshot 2024-10-17 at 2 03 54 PM](https://github.com/user-attachments/assets/61bd29e3-1141-4579-b3ad-ef9bdf5d81cb)
![Screenshot 2024-10-17 at 2 02 44 PM](https://github.com/user-attachments/assets/627bf777-e908-419f-b865-535df82e4ee8)


## How to test 🧪

- Checkout this PR (or just use the preview link)
- Verify the regressions in the "Screenshots" section are all fixed 🔧  

## As an Author I have considered 🤔
